README-comments.md

```diff
@@ -0,0 +1,268 @@
+
+```bash
+kubectl --namespace kube-system get deploy/tiller-deploy
+```
+
+```
+NAME            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+tiller-deploy   1         1         1            1           3m
+```
+
+Install the sandbox. Since Chirper only uses Cassandra, we're disabling the other services but you can leave them enabled by omitting the `set` flag if you wish.
+
+```bash
+helm install lightbend-helm-charts/reactive-sandbox --name reactive-sandbox --set elasticsearch.enabled=false,kafka.enabled=false,zookeeper.enabled=false
+```
+
+Verify that it is available (this takes a minute or two):
+
+```bash
+kubectl get deploy/reactive-sandbox
+```
+
+```
+NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+reactive-sandbox   1         1         1            1           1m
+```
+
+Now clone the Chirper sample referred to by the Lightbend Orchestration for Kubernetes project:
+
+```
+git clone https://github.com/longshorej/lagom-java-chirper-tooling-example.git chirper
+cd chirper
+```
+
+Add a namespace to the Container Registry to hold the images the Chirper build process will create:
+
+```
+bx plugin install container-registry -r Bluemix
+bx cr namespace-add chirper
+```
+
+If you were using Minikube you would invoke the 'sbt-native-packager' to create Docker images using `sbt docker:publishLocal`, but we're going to switch to publish to the Container Registry. For that we need to give `sbt` the registry location and namespace. You may already know the hostname of the Container Registry - there is one in each IBM Cloud region. It's the 'Container Registry' line output when you execute `bx cr info`.
+
+For the namespace, we're going to use 'chirper'.
+
+Add the following to the `build.sbt` file to the end of the definition of the `project` method around line 122:
+
+```
+  .settings(
+    dockerRepository := Some("registry.eu-gb.bluemix.net/chirper")
+  )
+```
+
+Note: I'm in the UK so my registry hostname has `eu-gb` in its name. Yours may differ.
+
+Now, when you execute:
+
+```
+$ bx cr login
+$ sbt docker:publish
+```
+
+the first command will redirect the local Docker engine to the Container Registry, and the second command will build the code and publish the images there.
+
+You can check the images have uploaded by viewing the [IBM Cloud console page your private registry](https://console.bluemix.net/containers-kubernetes/registry/private). Which should look similar to:
+
+![Container Registry after publishing images](images/ContainerReg.png)
+
+### Deploy to Kubernetes
+Now we're ready to direct the Container Service (Kubernetes) to deploy the images and run the application.
+
+You will need to run the following to set up your environment for the deploy. Be sure to adjust the secret variables, cassandra service address and Container Registry host as necessary.
+
+```
+REGISTRY=registry.eu-gb.bluemix.net
+NAMEPSACE=chirper
+
+# Be sure to change these secret values
+
+chirp_secret="youmustchangeme"
```

Comment:
I think these should be `changeme` so that the user is forced to change them. Play/Lagom will crash on startup given this value, which is a good thing as we don't want people copy/pasting potentially insecure defaults.

Note that the Chirper readme still has `youmustchangeme` but it will be updated soon as well.

Reply:
ok cool - so `changeme` is a special value for Play/Lagom. I'll change to that.